locale/en/site.json

```diff
@@ -71,6 +71,9 @@
         "package-manager": {
             "link": "download/package-manager",
             "text": "Installing Node.js via package manager"
+        },
+        "shasums":{
+          "text": "Signed SHASUMS for release files"
```

Comment:
In this case I would also add the "link" property for consistency and use `{{site.download.shasums.link}}` in the href attribute.